src/confcom/azext_confcom/security_policy.py

```diff
@@ -502,6 +526,12 @@ def populate_policy_content_for_all_images(
                             }
                         image.set_user(user)
 
+                if self._fragment_contents and self.should_eliminate_container_covered_by_fragments(image):
+                    # these containers will get taken out later in the function
+                    # since they are covered by a fragment
+                    print("Container covered by fragment: ", image_name)
```

Comment:
just checking, do you mean to keep this print?

Reply:
yes but it would be better as a logger message